mrp_bom_version/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    active_draft = fields.Boolean(
```

Comment:
You do it the load/save methods of the config model. See https://github.com/OCA/server-tools/pull/76/files#diff-30a60a6e49d7e65247cfbf6c6911e349R73 (with the recommendations)